### ISSUE TITLE
[BUG]: Folia jar not working

### ISSUE THREAD

**Issue by Bafbi** [TheNewEconomy/EconomyCore#113]
### TNE Version

0.1.3.1

### Spigot Version

Folia version 1.20.6-5-master@d797082

### Your config.yml

default

### Your latest server log

https://mclo.gs/gS4XKDD

### What part of TNE is the bug affecting?

Won't Startup

### Is your server using online mode?

Yes

### Bug description

First of all the folia jar is not marked as supported because the plugin.yml of the Folia module is overwrite by the paper-plugin.yml of the Paper module, after that you use the Bukkit scheduler which is not supported. 

### How to reproduce

Patch the paper-plugin.yml then launch the server with the plugin.

**Comment by creatorfromhell** [comment]
> ### TNE Version
> 0.1.3.1
> 
> ### Spigot Version
> Folia version 1.20.6-5-master@d797082
> 
> ### Your config.yml
> default
> 
> ### Your latest server log
> https://mclo.gs/gS4XKDD
> 
> ### What modules are you using?
> _No response_
> 
> ### What part of TNE is the bug affecting?
> Won't Startup
> 
> ### Is your server using online mode?
> Yes
> 
> ### Bug description
> First of all the folia jar is not marked as supported because the plugin.yml of the Folia module is overwrite by the paper-plugin.yml of the Paper module, after that you use the Bukkit scheduler which is not supported.
> 
> ### How to reproduce
> Patch the paper-plugin.yml then launch the server with the plugin.

Will look at this when I get a chance, currently have some reports ahead of this. Paper plugin override probably happened when the code was overhauled with paper support, so appreciate that aspect.

**Comment by creatorfromhell** [comment]
> ### TNE Version
> 0.1.3.1
> 
> ### Spigot Version
> Folia version 1.20.6-5-master@d797082
> 
> ### Your config.yml
> default
> 
> ### Your latest server log
> https://mclo.gs/gS4XKDD
> 
> ### What modules are you using?
> _No response_
> 
> ### What part of TNE is the bug affecting?
> Won't Startup
> 
> ### Is your server using online mode?
> Yes
> 
> ### Bug description
> First of all the folia jar is not marked as supported because the plugin.yml of the Folia module is overwrite by the paper-plugin.yml of the Paper module, after that you use the Bukkit scheduler which is not supported.
> 
> ### How to reproduce
> Patch the paper-plugin.yml then launch the server with the plugin.

TLDR: The paper plugin file override also overrode the plugin main file, and then also the adventure update kinda broke some stuff, but tested this one and it seems to work.

[Folia-0.1.3.1-RELEASE-1.zip](https://github.com/user-attachments/files/16482949/Folia-0.1.3.1-RELEASE-1.zip)

**Comment by creatorfromhell** [comment]
> > ### TNE Version
> > 0.1.3.1
> > ### Spigot Version
> > Folia version 1.20.6-5-master@d797082
> > ### Your config.yml
> > default
> > ### Your latest server log
> > https://mclo.gs/gS4XKDD
> > ### What modules are you using?
> > _No response_
> > ### What part of TNE is the bug affecting?
> > Won't Startup
> > ### Is your server using online mode?
> > Yes
> > ### Bug description
> > First of all the folia jar is not marked as supported because the plugin.yml of the Folia module is overwrite by the paper-plugin.yml of the Paper module, after that you use the Bukkit scheduler which is not supported.
> > ### How to reproduce
> > Patch the paper-plugin.yml then launch the server with the plugin.
> 
> Will look at this when I get a chance, currently have some reports ahead of this. Paper plugin override probably happened when the code was overhauled with paper support, so appreciate that aspect. However it's probably a similar issue with the error, as if you looked at the source there is a separate folia scheduler but it probably got mixed up in the realignment.

TLDR: The paper plugin file override also overrode the plugin main file, and then also the adventure update kinda broke some stuff, but tested this one and it seems to work.

[Folia-0.1.3.1-RELEASE-1.zip](https://github.com/user-attachments/files/16486385/Folia-0.1.3.1-RELEASE-1.zip)
